backend/src/v5/models/fileRefs.js

```diff
@@ -62,7 +64,13 @@ FileRefs.fetchFileStream = async (teamspace, model, extension, fileName) => {
 		return { readStream: stream, size: entry.size };
 	} catch {
 		logger.logError(`Failed to fetch file from ${entry.type}. Trying GridFS....`);
-		sendFileMissingError({ teamspace, model, collection, refId: entry._id, link: entry.link }).catch((err) => {
+		await sendEmail(FILE_MISSING, config.contact?.email, {
```

Comment:
Actually can you just remove this line (and any corresponding files)

This will be easier to clean up once the other issue is merged, otherwise I'd have to remember to go clean up the mailer.